optionals/auto_load_settings_file/XEH_preInit.sqf

```diff
@@ -0,0 +1,12 @@
+#include "script_component.hpp"
+
+ADDON = false;
+
+// frame after preInit, but before postInit
+0 spawn {
```

Comment:
I would've used `CBA_fnc_execNextFrame`, but the `EachFrame` handle is installed at postInit - many frames after preInit.